src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -873,5 +873,32 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
             Contracts.CheckValue(catalog, nameof(catalog));
             return new PriorTrainer(CatalogUtils.GetEnvironment(catalog), labelColumnName, exampleWeightColumnName);
         }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="catalog"></param>
+        /// <param name="options"></param>
+        /// <returns></returns>
+        public static LdSvmTrainer LdSvm(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog, LdSvmTrainer.Options options)
+            => new LdSvmTrainer(catalog.GetEnvironment(), options);
+
+        public static LdSvmTrainer LdSvm(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
+            string labelColumnName = DefaultColumnNames.Label,
```

Comment:
Why aren't you using `labelColumnName`, `featureColumnName`, and `exampleWeightColumnName` in the method body?